### DISCUSSION TITLE
Specify particular partitions on the Ola Index maintenance scripts

### DISCUSSION THREAD

**Discussion by samirdave50** [olahallengren/sql-server-maintenance-solution#482] [Questions & Answers]
Hi Ola,

Is there a way of specifying particular partitions to be reorganized rather than than let your script choose automatically?  I have very large tables and want to manually reorganize certain partitions on certain days, etc.   This is for SQL 2012 Enterprise.
Thanks
Samir

**Comment by olahallengren** [comment]
There is currently no option for this.
